virl2_client/models/lab.py

```diff
@@ -158,6 +160,11 @@ def __init__(
         Dictionary containing all annotations in the lab.
         It maps annotation identifier to `models.Annotation`.
         """
+        self._smart_annotations: dict[str, SmartAnnotation] = {}
+        """
+        Dictionary containing all smart annotations in the lab.
+        It maps smart annotations identifier to `models.SmartAnnotation`.
+        """
```

Comment:
In PyCharm at least, it shows you the comment below a variable the same way it shows you the docstring below a function definition

Reply:
tbh I have never seen comments being written bellow, anywhere, ever, but maybe I am just lucky avoiding this scourge.
Comments should always be written on top of the "upcoming" code, in case of a single line (like a variable) they can be written in-line... pycharm built different I guess